ext/net/01_net.js

```diff
@@ -100,10 +100,10 @@ class Conn {
   #writable;
 
   constructor(rid, remoteAddr, localAddr) {
-    ObjectDefineProperty(this, internalRidSymbol, {
-      enumerable: false,
-      value: rid,
-    });
+    if (internals.future) {
```

Comment:
Note: this change also takes care of `TlsConn` and `UnixConn` due to inheritance.